example/redisfailover/pmem.yaml

```diff
@@ -15,14 +14,17 @@ spec:
       - "--protected-mode"
       - "no"
   redis:
+    securityContext:
+      runAsNonRoot: False
     replicas: 3
-    image: redis_pmem # From https://github.com/pmem/redis
+    image: pmem/redis
     version: latest
     command:
       - "redis-server"
       - "/redis/redis.conf"
       - "--pmdir"
-      - "/data 100Mb"
+      - "/data"
+      - " 100Mb"
```

Comment:
What is the value that `--pmdir` takes? A single string or two of them? Has that changed?

If it is now two, then you can remove the leading space in `  100Mb`. If it is one, then splitting that string into two parameters is not correct.